src/kv.ts

```diff
@@ -79,7 +82,7 @@ export function parseKVString(input: string): KVPair {
       .trim()
       .replace(/\\([,\n])/gi, '$1');
 
-    if (!k || !v) {
+    if (!k || (!v && !allowEmptyValues)) {
```

Comment:
The problem with this approach is that It's not a label with no value, it's a label whose value is the empty string.

Reply:
> `FOO=,BAR=BAZ`
>
> It's reasonable to assume that some users would expect the same behavior as above, but others would expect a label FOO with a literal value of ,BAR=BAZ.

Wouldn't that require escaping the comma or using quotes, though? E.g `FOO=\,BAR=BAZ` or `FOO=",BAR=BAZ"`. Seeing it's a comma-separated list, that seems more reasonable don't you think?

Regardless, and so we're on the same page, we want to be able to do both `FOO,BAR=BAZ` _and_ `FOO=,BAR=BAZ`? Because I'd expect the first one to be an error case as there's no `=` and currently that's checked before the value. I believe this behaviour should persist.

An empty value should be fairly explicit as it tends to be an override or at least a conscious decision, but I'd like your input as well

Reply:
> Wouldn't that require escaping the comma or using quotes, though? E.g FOO=\,BAR=BAZ or FOO=",BAR=BAZ". Seeing it's a comma-separated list, that seems more reasonable don't you think?

Yes, this is why I think we should re-use some of the logic in #30 since it uses backtracking to handle escaping

> Regardless, and so we're on the same page, we want to be able to do both FOO,BAR=BAZ and FOO=,BAR=BAZ? Because I'd expect the first one to be an error case as there's no = and currently that's checked before the value. I believe this behaviour should persist.

Making the `=` mandatory seems like a good idea.

Reply:
Sorry for the delay, I've been quite busy these past weeks. I'll try my best to tackle this over the next weekend or as soon as I have some time on my hands!